OpenRA.Mods.Common/Scripting/Global/UserInterfaceGlobal.cs

```diff
@@ -44,7 +44,7 @@ public string Translate(string text, LuaTable table = null)
 					using (kv.Key)
 					using (kv.Value)
 					{
-						if (!kv.Key.TryGetClrValue(out string variable) || !kv.Value.TryGetClrValue(out object value))
+						if (!kv.Key.TryGetClrValue<string>(out var variable) || !kv.Value.TryGetClrValue<object>(out var value))
```

Comment:
No, we forgot to add this in a previous pull request, so I added here.

Reply:
In that case, the commit message should be changed since this doesn't seem to have anything to do with adding translation arguments to the Lua API.

However, I don't see why we'd want this at all? This is making the line more verbose!

Reply:
@abcdefg30 requested it.

Reply:
@abcdefg30 what do you think here?

Reply:
I requested this in the original PR as we usually use var over explicit types. I don't think that change here is too important though. If we want to keep it the commit message should be changed though.